packages/cli/src/generators/pull.js

```diff
@@ -0,0 +1,54 @@
+const colors = require('colors/safe');
+const debug = require('debug')('zapier:pull');
+const inquirer = require('inquirer');
+const path = require('path');
+const Generator = require('yeoman-generator');
+
+const maybeOverwriteFiles = async (gen) => {
+  const dstDir = gen.options.dstDir;
+  const srcDir = gen.options.srcDir;
+  for (const file of gen.options.sourceFiles) {
+    gen.fs.copyTpl(
+      gen.templatePath(path.join(srcDir, file)),
+      gen.destinationPath(path.join(dstDir, file)),
+      gen.options
+    );
```

Comment:
1. We only need `gen.fs.copy()` here because `copyTpl()` is for rendering a template.
2. `gen.templatePath()` and `gen.destinationPath()` are unnecessary because we're already dealing with absolute paths with `path.join()`.

```suggestion
    gen.fs.copy(path.join(srcDir, file), path.join(dstDir, file));
```